pkgs/misc/tmux-plugins/default.nix

```
@@ -109,6 +109,19 @@ in rec {
     };
   };
 
+  fingers = mkDerivation {
+    pluginName = "fingers";
+    version = "1.0.1";
+    src = fetchFromGitHub {
+      owner = "Morantron";
+      repo = "tmux-fingers";
+      rev = "d722ec7b3af18924f902a77c9e0beda5ca643f04";
```

```suggestion
      rev = version;
```
This will check out the tag corresponding to v1.0.1